man/os-release.xml

```diff
@@ -411,6 +411,15 @@
           determines the fallback hostname.</para></listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><varname>ARCHITECTURE=</varname></term>
+          <listitem><para>A string that specifies which CPU architecture the binaries require. The architecture
+          is not exactly in the format reported by <citerefentry><refentrytitle>uname</refentrytitle><manvolnum>2</manvolnum></citerefentry>
+          but uses the format as shown by <citerefentry><refentrytitle>hostnamectl</refentrytitle><manvolnum>1</manvolnum></citerefentry>.
+          An architecture should be specified in extensions images to prevent an incompatible host from loading it.
+          </para></listitem>
```

Comment:
Please conditionalize this a bit, i.e. say that this is an optional field and only applies to systems that are intended for a single architecture only. (because debian intends to support true multi-arch, where an single image might be for multiple archs actually)

Also say that this is redundant information, as the on GPT disk and images the same information can likely be derived from the partition type UUID the OS is stored in.

Also say that this is supposed to be the userspace architecture. Sparc Linux was traditionally a 64bit kernel with a 32bit userspace, and this here is strictly about userspace.

Reply:
Thanks, done, please check the wording again